modules/porous_flow/test/tests/jacobian/tests

```diff
@@ -1134,6 +1134,26 @@
     issues = '#6845'
     design = 'porous_flow/tutorial_09.md porous_flow/tutorial_02.md porous_flow/solvers.md porous_flow/nonlinear_convergence_problems.md PorousFlowDictator.md'
   [../]
+  [./esbc01]
+    type = 'PetscJacobianTester'
+    input = 'esbc01.i'
+    ratio_tol = 1E-6
+    difference_tol = 1E10
+    threading = '!pthreads'
+    requirement = "PorousFlow shall compute all Jacobian entries of all BCs"
+    issues = '#15742'
+    design = 'PorousFlowEnthalpySink.md'
+  [../]
+  [./esbc02]
+    type = 'PetscJacobianTester'
+    input = 'esbc02.i'
+    ratio_tol = 1E-6
```

Comment:
i reckon you can tighten this tolerance since the jacobian is zero

Reply:
✅